.github/workflows/perf_sysbench.yml

```diff
@@ -23,15 +23,27 @@ jobs:
       fail-fast: false
 
     steps:
-      - name: set PATH to GIT of the newer version 2.9.0
-        run: echo "/usr/local/git/bin" | tee -a $GITHUB_PATH
-      - name: cleanup workspace
-        run: docker run -w /source -v ${PWD}:/source -i centos:8 /bin/bash -c "chown -R $(id -u):$(id -g) * .[^.]*"
+#      TODO: migrate to v2 when Git 2.18 or higher would be available
       - uses: actions/checkout@v1
-      - name: test
-        env:
-          BENCH: 'sysbench'
-        uses: ./.github/actions/perf
+        with:
```

Comment:
I don't understand why you use `v1` here but `v2.3.4` below.

Reply:
It is because there is no supportable git on the machine, which is needed for submodule checkouts in `v2.3.4`